documentation/tutorials/kaggle_beginner_example_classification.ipynb

```diff
@@ -0,0 +1,2105 @@
+{
```

Comment:
<p>Watch your indentation here. The <a href="https://google.github.io/styleguide/pyguide.html" target="_blank">Google Python style guide</a> would allow either a 4-space hanging indent for all args, or you could align them with the <code>(</code> . e.g.</p><p> </p><pre class="ql-syntax" spellcheck="false">train_ds = tfdf.keras.pd_dataframe_to_tf_dataset(
    train_ds_pd,&nbsp;
    label=label,&nbsp;
    task=tfdf.keras.Task.CLASSIFICATION)
</pre><p> </p><p>or</p><p> </p><pre class="ql-syntax" spellcheck="false">train_ds = tfdf.keras.pd_dataframe_to_tf_dataset(train_ds_pd,
                                                 label=label,&nbsp;
                                                 task=tfdf.keras.Task.CLASSIFICATION)
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/decision-forests/pull/128/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>